fury/interactor.py

```diff
@@ -297,6 +361,7 @@ def SetInteractor(self, interactor):
         self.AddObserver("RightButtonReleaseEvent", self._process_event)
         self.AddObserver("MiddleButtonPressEvent", self._process_event)
         self.AddObserver("MiddleButtonReleaseEvent", self._process_event)
+        self.AddObserver("DoubleClickEvent", self._process_event)
```

Comment:
From what I could gather VTK doesn't truly support DoubleClickEvent by default, so this observer will never fire up. That's why people implemented the double click behavior from scratch like you just did (i.e. tracking clicks count).

So, you can remove this line and should still be working fine.